server/src/services/server.service.ts

```diff
@@ -147,7 +150,14 @@ export class ServerService implements OnEvents {
       serverStats.videos += usage.videos;
       serverStats.usage += usage.usage;
       serverStats.usageByUser.push(usage);
+
+      this.metricRepository.user.addToGauge(`immich.user.photos`, usage.photos, {},{"userName": user.userName})
+      this.metricRepository.user.addToGauge(`immich.user.videos`, usage.videos, {},{"userName": user.userName})
+      this.metricRepository.user.addToGauge(`immich.user.usage`, usage.usage, {},{"userName": user.userName})
     }
+    this.metricRepository.host.addToGauge(`immich.host.photos`, serverStats.photos)
+    this.metricRepository.host.addToGauge(`immich.host.videos`, serverStats.videos)
+    this.metricRepository.host.addToGauge(`immich.host.usage`, serverStats.usage)
```

Comment:
Btw, am I being stupid or you cannot send metrics without creating "sums"

Currently this keeps adding up the one photo I have. I can't use a Counter because it can decrease and the Gauge is adding stuff. 

I am very noob in Otel, I am used to send prometheus data with bash or python directly :)